### PR TITLE
permission documentation update

### PR DESCRIPTION
**Pull request by vsmk98**
This PR updates the permissions documentation to cover the v1 and v2 models as supported in the master branch of GoQuorum now.

### CODE REVIEW

File: docs/Reference/APIs/PermissioningAPIs.md [hunk 1]
```diff
@@ -1317,6 +1317,90 @@ Once majority approvals from network admin accounts is received, the node is mar
         "Action completed successfully"
         ```
 
+### `quorumPermission_transactionAllowed`
+
+This api checks if the account initiating the transction has sufficient permissions to execute the transaction
```
Comment (NicolasMassart) [inline docs/Reference/APIs/PermissioningAPIs.md:1322]:
```suggestion
This api checks if the account initiating the transaction has sufficient permissions to execute the transaction
```

File: docs/Concepts/Permissioning/PermissionsOverview.md [hunk 2]
```diff
@@ -2,9 +2,8 @@
 
 GoQuorum supports two network permissioning models.
 
-* [Basic network permissioning](BasicNetworkPermissions.md)
-    Controls which nodes can connect to a given node and also to which nodes the given node can dial out to.
+* [Basic network permissioning](BasicNetworkPermissions.md): Controls which nodes can connect to a given node and also to which nodes the given node can dial out to.
+* [Enhanced network permissioning](Enhanced/EnhancedPermissionsOverview.md): Caters for enterprise-level needs by having a **smart contract-based permissioning model**. This allows for significant flexibility to manage nodes, accounts, and account-level access controls. There are two versions of enhanced network permissioning as described below:
```
Comment (NicolasMassart) [inline docs/Concepts/Permissioning/PermissionsOverview.md:6]: maybe find a synonym for "Caters" that will not show up as food related as the first meaning in dictionaries, can be very confusing for non english readers (and for instance macOS look up only provides the first meaning. Perhaps something like "Meets enterprise level needs"?

File: docs/Reference/APIs/PermissioningAPIs.md [hunk 3]
```diff
@@ -1317,6 +1317,90 @@ Once majority approvals from network admin accounts is received, the node is mar
...
+
+        Response:
+
+        ```json
+        "true"
+        ```
+
+### `quorumPermission_connectionAllowed`
+
+This api checks if the given enode is permissioned to join network
```
Comment (NicolasMassart) [inline docs/Reference/APIs/PermissioningAPIs.md:1363]:
```suggestion
This api checks if the given enode is allowed to join network
```
Providing a synonym in the description helps to understand what the method does without using the same verb like what you already did on line 1373. Note that permission only appear as a noun in most dictionaries.

File: docs/Concepts/Permissioning/PermissionsOverview.md (hunk 2 above)
Comment (vsmk98) [reply docs/Concepts/Permissioning/PermissionsOverview.md]: sure. updated.

File: docs/Concepts/Permissioning/PermissionsOverview.md [hunk 4]
```diff
@@ -2,8 +2,8 @@
 
 GoQuorum supports two network permissioning models.
 
-* [Basic network permissioning](BasicNetworkPermissions.md): Controls which nodes can connect to a given node and also to which nodes the given node can dial out to.
-* [Enhanced network permissioning](Enhanced/EnhancedPermissionsOverview.md): Caters for enterprise-level needs by having a **smart contract-based permissioning model**. This allows for significant flexibility to manage nodes, accounts, and account-level access controls. There are two versions of enhanced network permissioning as described below:
+* **Basic network permissioning:** Controls which nodes can connect to a given node and also to which nodes the given node can dial out to. Refer [here](BasicNetworkPermissions.md) for further details.
```
Comment (NicolasMassart) [inline docs/Concepts/Permissioning/PermissionsOverview.md:5]: the previous version was better. link like `[here]()` or `[click this link]()` [should be avoided for multiple reasons ](https://medium.com/@heyoka/dont-use-click-here-f32f445d1021)

**Review by NicolasMassart — APPROVED** [review]

**Pull request merged**